scripts/observer/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import tinycolor from 'tinycolor2';
+import tokens from '@shopify/polaris-tokens';
+import {Box, Text, render, Color} from 'ink';
+
+const data = [
```

Comment:
@amrocha Here's the kind of data I'd expect from the other side, but pretty much anything less structured also would work, as long as the same amount of raw information is available.